src/languages/javascript.js

```diff
@@ -250,8 +250,8 @@ export default function(hljs) {
             begin: '(\\([^(]*' +
               '(\\([^(]*' +
                 '(\\([^(]*' +
-                '\\))?' +
-              '\\))?' +
+                '\\)\\s*)?' +
+              '\\)\\s*)?' +
```

Comment:
Not only spaces, I guess? Consider this:
```js
class Test {
  test(a = (3+2), b = 1) {
    console.log(a + b)
  }
}
```
Looks like `[^)]*` will do the job.

Reply:
Actually you also have to account for later expressions surrounded by ()... the new pattern tries to match:

```
(
.*(  
.*(
.*  
).*
).*
)
```

Where . means not `()`.